vulyk/models/stats.py

```diff
@@ -25,6 +29,7 @@ class WorkSession(Document):
     user = ReferenceField(User, reverse_delete_rule=CASCADE, required=True)
     task = ReferenceField(AbstractTask, reverse_delete_rule=CASCADE,
                           required=True)
+    task_type = StringField(max_length=50, required=True, db_field='taskType')
     answer = ReferenceField(AbstractAnswer, reverse_delete_rule=CASCADE)
 
     start_time = DateTimeField(required=True)
```

Comment:
If you look [here](https://github.com/mrgambal/vulyk/blob/master/vulyk/models/tasks.py#L24), you should notice we used name transformations to keep column and table names in camelCase format to adhere js conventions.
Although in WorkSession model and even [here and below](https://github.com/mrgambal/vulyk/blob/master/vulyk/models/user.py#L20) we have omitted these directives and meta.